db/pf-schema-X.Y.Z.sql

```diff
@@ -52,7 +52,7 @@ CREATE TABLE class (
   delay_by int(11) NOT NULL default 0,
   external_command varchar(255) DEFAULT NULL,
   PRIMARY KEY (security_event_id)
-) ENGINE=InnoDB;
+) ENGINE=InnoDB CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
```

Comment:
To anybody from @inverse-inc thinking of merging this PR, this will require internal discussions on the modifcation of the encoding of the database since this has very large concequences (external DBs, upgrade of existing databases, code that relies on a specific encoding)